src/java.base/share/classes/java/util/concurrent/ForkJoinPool.java

```diff
@@ -676,7 +676,10 @@ public class ForkJoinPool extends AbstractExecutorService {
      * Trimming workers. To release resources after periods of lack of
      * use, a worker starting to wait when the pool is quiescent will
      * time out and terminate if the pool has remained quiescent for
-     * period given by field keepAlive.
+     * period given by field keepAlive (default 60sec), which applies
+     * to the first timeout of a fully populated pool. Subsequent (or
```

Comment:
If I understand this correctly, it means that the entire pool needs to have no work (at all) before starting to decommission workers. Is that the case?